index.js

```diff
@@ -3232,6 +3232,55 @@
   S.range =
   def('range', {}, [$.Integer, $.Integer, $.Array($.Integer)], range);
 
+  //# groupBy :: (a -> a -> Boolean) -> Array a -> Array (Array a)
+  //.
+  //. Splits its list argument into a list of lists of equal,
+  //. adjacent elements. The equality of the elements is tested with
+  //. a curried function provided by the user in the first argument.
+  //. The behavior of groupBy aims to be equal to Haskell's groupby,
+  //. rather than Ramda's groupBy.
+  //.
+  //. The behavior of groupBy can be surprising for functions that
```

Comment:
Nit: Let's write behaviour with a "u" for consistency with the rest of the documentation.